packages/core/src/config.ts

```diff
@@ -81,6 +82,15 @@ import {
 } from './utils/syntax';
 import { loadTsconfig } from './utils/tsconfig';
 
+rsbuildLogger.override({
+  start: (message) => {
```

Comment:
Overriding start is not a good idea, how about using a hook like `onBeforeBuild`

Reply:
We cannot guarantee that logger.start will not be called from somewhere else

Reply:
I tried `onBeforeBuild` before, but when multiple environments presents. It will be called multiple times. I think I should re-implement something like `Building...` with Rspack `watchRun` and `done` hook in Rslib side.

Reply:
> onBeforeBuild is a global hook, are you registering plugins in global？

Might be.

Reply:
Rsbuild will not clear the logs before rebuilding, as this will likely cause useful logs to be cleared and make troubleshooting more difficult.

Reply:
I'd like to give a try on Rslib, hoping it fit with the simpler lib scenario. 😀

Reply:
In fact, Rsbuild early versions will clear the log by default, but we later found that this caused more trouble than benefit, so we canceled it. However, for Rslib scenarios it is still worth a try.

Reply:
Can't find an elegant way to archive this, I'm heading to close for now.

- `onBeforeBuild` will run after `Building...` and clear it, so it's not an option.
- `watchRun` is good, but can't find a way to register the plugin for only once. Register to the first environment only is also an ugly workaround.